db/models/models.js

```diff
@@ -10,6 +10,21 @@ const fetchTopics = () => {
     })
 }
 
+const fetchArticles = (sort_by='article_id') => {
+    const sqlString = 
+    `
+        SELECT articles.*, COUNT(comments.comment_id) as comment_count
+        FROM articles
+        LEFT JOIN comments
+        ON articles.article_id = comments.article_id
+        GROUP BY articles.article_id
+        ORDER BY ${sort_by} DESC;
```

Comment:
the articles should be sorted by date in descending order as per the ticket

Reply:
Oh. I understood. At first I thought it has to be queried with '?sort_by=created_at' but now see that 'created_at' has to be a default value